dll/win32/shell32/shlfileop.cpp

```diff
@@ -757,7 +773,22 @@ static DWORD SHNotifyCopyFileW(FILE_OPERATION *op, LPCWSTR src, LPCWSTR dest, BO
         return ERROR_SUCCESS;
     }
 
+#ifdef __REACTOS__
+    DWORD error = GetLastError();
+
+    if (error == ERROR_DISK_FULL && !(op->req->fFlags & (FOF_NOERRORUI | FOF_SILENT)))
+    {
+        CStringW strTitle, strText;
+        strTitle.LoadStringW(IDS_COPYERRORTITLE);
+        strText.LoadStringW(IDS_DISKFULL);
+
+        MessageBoxW(op->req->hwnd, strText, strTitle, MB_ICONERROR);
+    }
```

Comment:
But also I'm thinking... any reason to place warning message box right in these functions? They both return error code, so maybe it's possible to check for that error in other function(s) which call these two.

Reply:
i thought about that, but both functions check input flags FOF_NOERRORUI | FOF_SILENT as validate_operation do